### PR TITLE
Replace resources page

### PR DESCRIPTION
**Pull request by mosabua**
Much improved replacement for https://trino.io/resources.html :

- Remove hardcoded old page
- Add content to data file
- Add layouts and includes for data driven pages
- Add header navigation
- Add images
- Add wording and links as need

### CODE REVIEW

File: ecosystem/add-ons-intro.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Add-ons provide support for additional use cases and scenarios for your
```
Comment (colebow) [inline ecosystem/add-ons-intro.md:1]: Maybe "provide additional utility and expand on what Trino has to offer" or something like that? "additional use cases and scenarios" doesn't really mean much.

File: ecosystem/clients-intro.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Clients allow you to connect to Trino, submit SQL queries, and receive the
+results. Clients are full-featured applications or libraries and drivers that
+allow you to connect to your custom application. Clients simple command line
```
Comment (colebow) [inline ecosystem/clients-intro.md:3]: "Clients simple command line" isn't right.

File: ecosystem/solution.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Solutions
+show_hero: true
+show_pagenav: true
+---
+
+Solutions integrate Trino to create a full-featured product for specific use
+cases. They can simplify Trino management, provide additional features and
+extensions such as security and access control enhancement, or adapt Trino to
```
Comment (colebow) [inline ecosystem/solution.md:10]:
```suggestion
extensions, such as security and access control enhancement, or adapt Trino to
```

File: ecosystem/solutions-intro.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Solutions integrate Trino to create a full-featured product for specific use
+cases. They can simplify Trino management, provide additional features and
+extensions such as security and access control enhancement, or adapt Trino to
```
Comment (colebow) [inline ecosystem/solutions-intro.md:3]:
```suggestion
extensions, such as security and access control enhancement, or adapt Trino to
```

We're duplicating this text in solution.md and solutions-intro.md. Is that correct?

Comment (mosabua) [reply ecosystem/solutions-intro.md:3]: Opps .. fixed

File: _includes/toolslist.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      <div class="row">
+        <div class="col-md-3 col-s-12 center-image">
+          {% if tool.logo == nil %}
+          {{tool.name}}
+          {% else %}
+          <img src="{{tool.logo}}" alt="{{tool.name}}" title="{{tool.name}}" class="img-fluid img-padding">
+          {% endif %}
+        </div>
+        <div class="col-md-9 col-s-12">
+<div markdown="1">### {{tool.name}}
```
Comment (mosabua) [inline _includes/toolslist.html:18]: This weird markdown and html mixture is necessary to make the right hand nav work automatically

**Comment by mosabua** [comment]
Demoed to @martint and addressed feedback. Also got feedback from @dain and addressed that. 

From our discussions on slack this is essentially approved now and I am going to merge in the interest of getting this huge improvement live. Also just to clarify we will follow this up with adding more clients and add-ons soon in separate PRs. And another PR will add a whole separate section for data sources that basically will refer to all the connector that are part of trinodb and beyond. 

This will be a great asset overall for showing the success of the ecosystem around the project and the momentum of support and collaboration with other communities.

**Comment by nineinchnick** [comment]
You now can't search for text on this page and have to visually inspect all logos. Won't that affect SEO?

**Pull request merged**